generatederrors.go

```diff
@@ -0,0 +1,676 @@
+// This source file is part of the EdgeDB open source project.
+//
+// Copyright 2020-present EdgeDB Inc. and the EdgeDB authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This file is auto generated. Do not edit!
+
+package edgedb
+
+import "fmt"
+
+const (
+	internalServerErrorCode uint32 = 0x01_00_00_00
```

Comment:
Why do these start with lowercase? Wouldn't it make sense to make them "public"? And they would also look "better" when started with an uppercase letter, IMO

Reply:
The code values are not part of the public API currently. Error values do not have a code method or field. Codes are only an internal detail currently. What does making the codes public help?

Reply:
> What does making the codes public help?

You're right, my mistake. We'll be actually removing error codes soon so exporting them is a bad idea.